### PR TITLE
chore: bump critical dependencies

### PR DESCRIPTION
**Pull request by edalholt**
Fixes vulnerabilities in Vite, Axios and Cypress

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -37,16 +37,15 @@
...
     "@types/react-dom": "^18.0.9",
     "@vitejs/plugin-react-swc": "^3.2.0",
     "@vitest/coverage-istanbul": "^0.28.3",
     "@vitest/ui": "^0.28.3",
-    "cypress": "^12.10.0",
+    "cypress": "^12.17.2",
     "jsdom": "^21.1.0",
     "typescript": "^4.9.3",
-    "vite": "^4.0.0",
+    "vite": "^4.4.7",
```
Comment (Xtrah) [inline frontend/package.json:48]:
```suggestion
    "vite": "^4.4.9",
```

File: frontend/package.json [hunk 2]
```diff
@@ -37,16 +37,15 @@
...
   "devDependencies": {
     "@testing-library/react": "^13.4.0",
-    "@types/axios": "^0.14.0",
     "@types/react": "^18.0.26",
     "@types/react-dom": "^18.0.9",
     "@vitejs/plugin-react-swc": "^3.2.0",
     "@vitest/coverage-istanbul": "^0.28.3",
     "@vitest/ui": "^0.28.3",
-    "cypress": "^12.10.0",
+    "cypress": "^12.17.2",
```
Comment (Xtrah) [inline frontend/package.json:45]:
```suggestion
    "cypress": "^12.17.3",
```

File: frontend/package.json
Comment (Xtrah) [inline frontend/package.json:1]: Should also consider updating more stuff here, such as mantine and TypeScript

**Review by Xtrah — APPROVED** [review]
LGTM, but might as well update to latest if we're updating at all? 😁

File: frontend/package.json
Comment (edalholt) [reply frontend/package.json:1]: Yep, agree
(And those versions WAS the latest when this pr was created 🫠)

**Pull request merged**